src/util/orm.py

```diff
@@ -476,31 +616,72 @@ def create(self, values, **kw):
         if self._size:
             raise ValueError("`create` can only called on empty `browse_record` objects.")
 
-        ids = []
-        size = len(values)
-        it = chunks(values, self._chunk_size, fmt=list)
+        if self._strategy == "multiprocessing" and not multi:
```

Comment:
As discussed in DM: we should have two private implementations. The current one under `_create` and the new one in `_create_multiplrocessing` we call each depending on the options.